src/components/Navigation/index.tsx

```diff
@@ -61,8 +67,8 @@ const Navigation: React.FC<INavigation> = ({ setSidebar }) => {
         </ul>
       </StyledNavigationItems>
       <Flex align={'center'}>
-        <StyledConnectWalletButton>
-          Connect
+        <StyledConnectWalletButton onClick={readyToTransact}>
+          {address ? `${address?.slice(0, 5)}...` : 'Connect'}
```

Comment:
This is nice :) But we can probably use [this](https://github.com/aragon/client/blob/master/src/web3-utils.js#L256-L282) utility!

Reply:
Fixed in c36207998e0878960b97d5673b755d1ff8cd2869